extensions/ArchLinux/modules/skins/vector.less

```
@@ -115,12 +115,31 @@ body.skin-vector-2022 {
     display: block;
   }
 
-  // Align the TOC button and menu
-  #vector-toc-collapsed-button {
-    margin-left: 0;
+  // Fix style and layout of main menu in the left sidebar
+  @media screen {
+    /* note: #vector-main-menu alone is not unique as of MW 1.40 */
```

Use LESS style comment syntax and uppercase "NOTE" since some text editors highlight it.

```suggestion
    // NOTE: #vector-main-menu alone is not unique as of MW 1.40
```